plugins/behaviors/modal/Modal.d.ts

```diff
@@ -36,7 +38,7 @@ declare namespace Modal {
     }
 }
 
-declare class Modal extends Phaser.Events.EventEmitter {
```

Comment:
The class in [Modal.js](https://github.com/rexrainbow/phaser3-rex-notes/blob/bdfd8834852b53e072d85bec74a713999374b7ef/plugins/behaviors/modal/Modal.js#LL12C26-L12C26) extends OpenCloseTransition.

However, in the types, instead of OpenCloseTransition, Phaser.Events.EventEmitter is being used, which leads to a TypeScript error when trying to access Modal.scene.

Would it be possible to use OpenCloseTransition in the types as well, instead of Phaser.Events.EventEmitter?